src/asset-swapper/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -343,6 +345,46 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumerBase {
             };
         }
 
+        // OTC orders
```

Comment:
[no action required]
`getCalldataOrThrowAsync` needs to be refactored at some point lol

Reply:
yeah this is getting to be a massive function